pkg/rid/cockroach/identification_service_area.go

```diff
@@ -151,13 +156,26 @@ func (c *ISAStore) DeleteISA(ctx context.Context, isa *ridmodels.IdentificationS
 				identification_service_areas
 			WHERE
 				id = $1
-			AND
-				owner = $2
 			AND
 				updated_at = $3
 			RETURNING %s`, isaFields)
 	)
-	return c.processOne(ctx, deleteQuery, isa.ID, isa.Owner, isa.Version.ToTimestamp())
+	return c.processOne(ctx, deleteQuery, isa.ID, isa.Version.ToTimestamp())
+}
+
+// UnsafeDeleteISA deletes the IdentificationServiceArea identified by "id". It
+// should only be called in a transaction that verifies the version.
+// Returns the delete IdentificationServiceArea and all Subscriptions affected by the delete.
+func (c *ISAStore) UnsafeDeleteISA(ctx context.Context, isa *ridmodels.IdentificationServiceArea) (*ridmodels.IdentificationServiceArea, error) {
```

Comment:
Yup, this is just a first pass to spur discussion. I'd say let's keep the comments focused on the approach and once we agree I'll circle back and cleanup the implementation

Reply:
Sounds good, and I think a key feature of the approach is where which functions are defined.  There are (at least) three places where UnsafeDeleteISA could be declared, and I would suggest that the Cockroach ISAStore itself is probably not the best place to put it unless it's incidental to Cockroach-only functionality (I don't think that's the intention here, but please correct me otherwise)